File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/features/Lifter.java
package org.firstinspires.ftc.teamcode.features;

import org.firstinspires.ftc.teamcode.internals.features.Feature;
import org.firstinspires.ftc.teamcode.internals.hardware.Devices;

/**
 * This is a test the robot screw lifter.
 * <p>
 * Connections: A motor in port 0 and a controller.
 * <p>
 * Controls: Move the left stick up and down to control the motor.
 */
public class Lifter extends Feature {
    @Override
    public void loop() {
        double motorSpeed = -Devices.controller1.getLeftStickY();
        Devices.motor0.setSpeed(motorSpeed);
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/features/Lifter.kt
package org.firstinspires.ftc.teamcode.features

import org.firstinspires.ftc.teamcode.internals.features.Feature
import org.firstinspires.ftc.teamcode.internals.hardware.Devices

/**
 * This is a test the robot screw lifter.
 *
 *
 * Connections: A motor in port 0 and a controller.
 *
 *
 * Controls: Move the left stick up and down to control the motor.
 */
class Lifter : Feature() {
    override fun loop() {
        val motorSpeed = (Devices.controller1.rightTrigger - Devices.controller1.leftTrigger)
        Devices.motor3.speed = motorSpeed
    }
}
